Jamfile

```diff
@@ -766,6 +770,7 @@ lib torrent
 
 	<dht>on:<source>src/kademlia/$(KADEMLIA_SOURCES).cpp
 	<dht>on:<source>ed25519/src/$(ED25519_SOURCES).cpp
+	<dht>on:<source>src/hasher512.cpp
```

Comment:
I wonder if this extra build logic is worth it. I assume the code in that .cpp file could just as well have ifdefs around it for the DHT being enabled